guides/v2.3/graphql/queries.md

```diff
@@ -1,35 +1,167 @@
 ---
 group: graphql
-title: Searches and pagination in GraphQL
+title: Queries
+redirect_from:
+  - /guides/v2.3/graphql/search-pagination.html
 ---
 
-A GraphQL search query can contain the following components:
+A GraphQL query retrieves data from the Magento server in a similar manner as a REST GET call. However, Magento GraphQL queries are not meant to be as comprehensive as their REST counterparts. The main purpose of the Magento GraphQL queries is to allow a mobile app or browser to render a wide variety of information, including the following:
```

Comment:
`are not meant to be as comprehensive as their REST` this seems to be inaccurate.  GraphQL must cover all storefront scenarios covered with REST and make it possible to query more data in one request.
Long-term the main difference is that GraphQL will support store-front use cases while REST will support admin use cases.